source/module_base/module_mixing/broyden_mixing.cpp

```diff
@@ -140,7 +140,7 @@ void Broyden_Mixing::tem_cal_coef(const Mixing_Data& mdata, std::function<double
         double* work = new double[ndim_cal_dF];
         int* iwork = new int[ndim_cal_dF];
         char uu = 'U';
-        int info;
+        int info = 0;
```

Comment:
Please add initializing for line 140 and 141 or drop the change in line 143.
They are output of line 144